test/test_client.py

```diff
@@ -828,13 +823,8 @@ def test_close_stops_kill_cursors_thread(self):
         client.close()
         self.assertTrue(client._kill_cursors_executor._stopped)
 
-        # Reusing the closed client should restart the thread.
-        client.admin.command('ping')
-        self.assertFalse(client._kill_cursors_executor._stopped)
-
-        # Again, closing the client should stop the thread.
-        client.close()
-        self.assertTrue(client._kill_cursors_executor._stopped)
+        # Reusing the closed client should raise an InvalidOperation error.
+        self.assertRaises(InvalidOperation, client.admin.command, 'ping')
```

Comment:
I'm not sure we need this if we are already testing that it raises an InvalidOperation exception in test_client.test_close

Reply:
This line becomes meaningful once we add this:
```python
self.assertRaises(InvalidOperation, client.admin.command, 'ping')
# Thread is still stopped.
self.assertTrue(client._kill_cursors_executor._stopped)
```

Reply:
Done.